pkg/commands/artifact/run.go

```diff
@@ -252,16 +248,6 @@ func Run(ctx context.Context, opts flag.Options, targetKind TargetKind) (err err
 	r := NewRunner(ctx, scanSettings)
 	defer r.Close(ctx)
 
-	if !r.CacheUsed() && scanSettings.CacheUsed {
-		// re-cache rules
-		if opts.ScanOptions.Force && !opts.ScanOptions.Quiet {
-			outputhandler.StdOutLogger().Msgf("Caching rules")
-		}
-		if err = settings.RefreshRules(scanSettings, opts.ExternalRuleDir, opts.RuleOptions, formatFoundLanguages(inputgocloc.Languages)); err != nil {
-			return err
-		}
-	}
```

Comment:
Not sure I understand the changes in this file. This is removing the work I did to cache the bearer-rules. Is it a git / merge issue?